src/server/analytic_services/prometheus_reports/base_prometheus_report.js

```diff
@@ -32,6 +32,15 @@ class BasePrometheusReport {
     export_metrics() {
         return this.registry.metrics();
     }
+
+    get_metric(name) {
+        const metric_name = this.get_prefixed_name(name);
+        return this.registry.getSingleMetric(metric_name);
+    }
+
+    reset_metrics() {
+        return this.registry.resetMetrics();
+    }
```

Comment:
From the review in the last communitee meeting, I understand that these functions are used only accessed from tests.
In that case, I think it is preferable to define them as util function in the tests framework itself, so they are not part of the base report interface.

For exmple:
```
function reset_metrics(report) {
    report.registry.resetMetrics();   
}
```

In this case, the code base uses the reports as a blackbox while the test code uses the reports as a white box.

Reply:
ok